CONTRIBUTING.md

```diff
@@ -91,15 +91,19 @@ Desk](https://jira.niis.org/servicedesk/customer/portal/1) and
 Before submitting a bug:
 
 - Make sure that you are using the latest version.
-- Check the [Knowledge Base](https://confluence.niis.org/display/XRDKB) and [X-Road community Slack](https://jointxroad.slack.com/) for a list of common questions and problems.
+- Check the [Knowledge Base](https://confluence.niis.org/display/XRDKB) for a list of common questions and problems.
+- Check the [X-Road community Slack](https://jointxroad.slack.com/) for a list of common questions and problems.
```

Comment:
Maybe I would still change the wording on this to something like:
"- Ask others on [X-Road community Slack](https://jointxroad.slack.com/) in case this is a known issue or working as expected."

Reply:
I updated the phrase, but I included checking Slack for previous questions on the same topic too. Since many questions have already been answered on Slack, it's useful to do a search there too.